api/v1alpha1/restore_types.go

```diff
@@ -17,6 +17,8 @@ type RestoreSpec struct {
 	KeepJobs      *int           `json:"keepJobs,omitempty"`
 	// Tags is a list of arbitrary tags that get added to the backup via Restic's tagging system
 	Tags []string `json:"tags,omitempty"`
+	// ResourceRequirementsTemplate describes the compute resource requirements (cpu, memory, etc.)
+	ResourceRequirementsTemplate *corev1.ResourceRequirements `json:"resourceRequirementsTemplate,omitempty"`
```

Comment:
Can you rename this property also in the other Job types (except Schedule)?
```suggestion
	// Resources describes the compute resource requirements of the containers in the scheduled Pod
	Resources *corev1.ResourceRequirements `json:"resourceRequirementsTemplate,omitempty"`
```